app/models/katello/input_template_renderer.rb

```diff
@@ -0,0 +1,7 @@
+class InputTemplateRenderer
+  include UnattendedHelper
+
+  def errata(id)
+    Katello::Erratum.find_by_errata_id(id).attributes.slice!('errata_id', 'created_at', 'updated_at')
```

Comment:
You might use:
```
Katello::Erratum.with_identifiers(id)
```

instead, as this will handle db ids (5) and errata ids (RHSA-2016:3)